### PR TITLE
address nits, fixes #7

### LINKED ISSUE #7: Minor nits
Nits for the next go-round:

1. The document needs "updates" added to the document metadata
2. The title is far more narrow than document content (which does more than simply adding a filename extension)
3. Don't include citations in Abstracts, and give documents some kind of description (e.g. `This document updates RFC 4329, "Scripting Media Types".`)

_cc @linuxwolf at his request_

### PR DESCRIPTION
**Pull request by linuxwolf**
* Changed the title to something a little more broad
* Dereferenced the abstract reference
* Updated metadata for Updates (and Area)
* Corrected a couple of other EcmaScript -> ECMAScript instances

### CODE REVIEW

File: javascript-mjs/draft.md [hunk 1]
```diff
@@ -86,7 +87,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 # Registration
 
-The ECMAScript media types are to be updated to point to a non-vendor specific standard undated specification of ECMAScript. In addition, a new file extension of .mjs is to be added to the list of file extensions with the restriction that it must correspond to the Module grammar of {{ECMA-262}}. Finally, the {{HTML}} specification is using text/javascript as the default media type of EcmaScript when preparing script tags; therefore, text/javascript has been moved intended usage from OBSOLETE to COMMON.
+The ECMAScript media types are to be updated to point to a non-vendor specific standard undated specification of ECMAScript. In addition, a new file extension of .mjs is to be added to the list of file extensions with the restriction that it must correspond to the Module grammar of {{ECMA-262}}. Finally, the {{HTML}} specification is using text/javascript as the default media type of  when preparing script tags; therefore, text/javascript has been moved intended usage from OBSOLETE to COMMON.
```
Comment (bmeck) [inline javascript-mjs/draft.md]: "the default media type of  when preparing script tags", was this meant to have ECMAScript?

Comment (linuxwolf) [inline javascript-mjs/draft.md]: I really thought I was more careful than this.  Fixed in latest commit.

If you're happy, I'd like to squash it down before taking merging the PR.

Comment (bmeck) [inline javascript-mjs/draft.md]: doh, merged before I saw this

Comment (linuxwolf) [inline javascript-mjs/draft.md]: No worries (-:

**Pull request merged**